tjdashboard/code/administrator/sql/install.mysql.utf8.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS `#__tj_dashboard` (

```

Comment:
Can you create this SQL fresh.. why is there no alias column here ??

Also, where is the widgets table ?

Reply:
Done